api/types/oidc.go

```diff
@@ -226,11 +222,6 @@ func (o *OIDCConnectorV3) SetIssuerURL(issuerURL string) {
 	o.Spec.IssuerURL = issuerURL
 }
 
-// SetRedirectURL sets client secret to some value
-func (o *OIDCConnectorV3) SetRedirectURL(redirectURL string) {
```

Comment:
Do we want to update the version of OIDCConnector to V4?

Reply:
I don't see a reason too, how come?